dev-libs/amdgpu-pro-opencl/amdgpu-pro-opencl-20.10.1048554.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit unpacker multilib-minimal
+
+SUPER_PN='amdgpu-pro'
+MY_PV=$(ver_rs 2 '-')
+
+DESCRIPTION="Proprietary OpenCL implementation for AMD GPUs"
+HOMEPAGE="https://www.amd.com/en/support/kb/release-notes/rn-amdgpu-unified-linux-20-10"
+SRC_URI="${SUPER_PN}-${MY_PV}-ubuntu-18.04.tar.xz"
+
+LICENSE="AMD-GPU-PRO-EULA"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RESTRICT="bindist mirror fetch strip"
+
+BDEPEND="dev-util/patchelf"
+COMMON=">=virtual/opencl-3"
+DEPEND="${COMMON}"
+RDEPEND="${COMMON}
```

Comment:
`COMMON` what? Rename to `COMMON_DEPEND` that's widely used.

Reply:
Ok